UI/login_screen.R

```diff
@@ -23,7 +23,8 @@ output$screen <- renderUI({
                  column(width = 12,
                         tags$h3("User ID", class = "mt-7")),
                  column(width = 12,
-                        textInput("name", NULL))  # Text input to enter the name.
```

Comment:
Do you think we can remove the login screen completely?

Reply:
I could try it and let  you know.

Reply:
Yes, we can get rid of the login screen